testing/testing_lib.sh

```diff
@@ -46,12 +46,36 @@ function run_jobmanager() {
     local image_tag="$(image_tag "$dockerfile")"
     local image_name="$(image_name "$image_tag")"
 
+    echo >&2 "Running ${image_tag} jobmanager..."
+
+    # Prints container ID
+    docker run \
+        --rm \
+        --detach \
+        --name "jobmanager" \
+        --network "$NETWORK_NAME" \
+        --publish 6123:6123 \
+        --publish 8081:8081 \
+        -e JOB_MANAGER_RPC_ADDRESS="jobmanager" \
+        "$image_name" \
+        jobmanager
+}
+
+function run_jobmanager_non_root() {
+    local dockerfile="$1"
+
+    local image_tag="$(image_tag "$dockerfile")"
+    local image_name="$(image_name "$image_tag")"
+
+    echo >&2 "Running ${image_tag} jobmanager as non-root..."
+
     # Prints container ID
     docker run \
         --rm \
         --detach \
         --name "jobmanager" \
         --network "$NETWORK_NAME" \
+        --user 1234 \
```

Comment:
Why it is `1234` Given that the Dockerfile script pins it to `9999`?

Reply:
This is intentional, to ensure that it will run successfully as an arbitrary user, not just `flink (9999)`. If this passes, there should also be no problem running as `9999`.

Reply:
Got it, thanks for the explanation.